docker-compose.yaml

```diff
@@ -0,0 +1,36 @@
+volumes:
+  ollama:
+
+networks:
+  ollama-docker:
+
+services:
+  ollama:
+    volumes:
+      - ./ollama/ollama:/root/.ollama
+    container_name: ollama
+    tty: true
+    image: ollama/ollama:latest
+    ports:
+      - 11434:11434
+    networks:
+      - ollama-docker
+    deploy:
+      resources:
+        reservations:
+          devices:
```

Comment:
after reading your PR, it's the compose only for nvdia
so rename it to `docker-compose.nvdia.yml`
And add one for non nvidia

Reply:
done